plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableHandle.java

```diff
@@ -32,26 +34,31 @@
     private final String tableName;
     private final TableType tableType;
     private final Optional<Long> snapshotId;
+    private final byte[] serializedSchema;
```

Comment:
I think there already was an idea to add schema to IcebergTableHandle and it was rejected (?) for some reason.

@phd3 do you remember?